lib/elixir/pages/anti-patterns/process-anti-patterns.md

```diff
@@ -4,7 +4,79 @@ This document outlines anti-patterns related to processes and process-based abst
 
 ## Code organization by process
 
-TODO.
+* **Problem:** This anti-pattern refers to code that is unnecessarily organized by processes. A process itself does not represent an anti-pattern, but it should only be used to model runtime properties (e.g., concurrency, access to shared resources, and event scheduling). When a process is used for code organization, it can create bottlenecks in the system.
+
+* **Example:** An example of this anti-pattern, as shown below, is a library that implements arithmetic operations (e.g., add, subtract) by means of a `GenSever` process. If the number of calls to this single process grows, this code organization can compromise the system performance, therefore becoming a bottleneck.
```

Comment:
```suggestion
* **Example:** An example of this anti-pattern, as shown below, is a library that implements arithmetic operations (such as add, subtract) by means of a `GenSever` process. If the number of calls to this single process grows, this code organization can compromise the system performance, therefore becoming a bottleneck.
```

Reply:
@josevalim (such as `add` and `subtract`) ?

Reply:
> (such as `add` and `subtract`) ?

I think that's better 👍